person-service/src/main/kotlin/com/thehuginn/repository/PersonRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.thehuginn.repository
+
+import com.thehuginn.domain.Person
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepositoryBase
+import jakarta.enterprise.context.ApplicationScoped
+import java.util.UUID
+
+@ApplicationScoped
+class PersonRepository : PanacheRepositoryBase<Person, UUID> {
+
+    fun getById(id: UUID) = findById(id) ?: throw NoSuchElementException()
+
+    fun getByName(firstName: String, lastName: String): Person {
+        return find("firstName = ?1 and lastName = ?2", firstName, lastName).firstResult()
```

Comment:
toto nebude fungovat uplne spravne, ak sa posle iba priezvisko napr. Taktiez {first,last}Name urcite mozu byt nullable, takze uz od queryParam by mali prist ako `String?`.
Lepsie je to spravit trochu dynamicky. Mozno nieco ako:

val query = StringBuilder("1=1")
firstName?.let { query.append(" and firstName = ?1") }
lastName?.let {query.append(" and lastName = ?2") }

return find(query.toString(), firstName, lastName).range(0, 50).list() a vracal by som list osob